dist/js/mapBackground.js

```diff
@@ -0,0 +1,31 @@
+var officeCoords = [60.1687025, 24.9341899];
+     
+function showGoogleMaps() {
+ 
+  var newOfficePosition = new google.maps.LatLng(officeCoords[0], officeCoords[1]);
+  var mapCenterPointPosition = new google.maps.LatLng(
+    officeCoords[0] + 0.002, 
+    officeCoords[1]
+  );
+
+  var mapOptions = {
+      zoom: 16, // initialize zoom level - the max value is 21
+      streetViewControl: false, // hide the yellow Street View pegman
+      scaleControl: false, // disable users from zooming the Google Map
+      mapTypeId: google.maps.MapTypeId.ROADMAP,
```

Comment:
That or HYBRID would be perhaps better than the current one?

### Hybrid
![screen shot 2015-03-18 at 16 34 13](https://cloud.githubusercontent.com/assets/4402654/6710951/a88c0562-cd8c-11e4-877d-5b7870792b86.png)

### Satellite
![screen shot 2015-03-18 at 16 34 22](https://cloud.githubusercontent.com/assets/4402654/6710955/b31e88c4-cd8c-11e4-9ea4-fa5992ece91f.png)